src/screens/AddSiteScreen.js

```diff
@@ -163,6 +179,21 @@ class AddSiteScreen extends Component {
                             {location}
                         </Text>
 
+                        <Button
+                            large
+                            rounded={true}
+                            onPress={this.onClickIAmHere}
+                            buttonStyle={buttonStyle}
+                            icon={{name: 'bullseye', type: 'font-awesome'}}
+                            title={here_now}
+                        >
+                            {campsite.upload}
+                        </Button>
+
+                        <Text h3
+                              style={headerTitle}
+                        >- or -</Text>
+
```

Comment:
I swapped the order of the button and the text input fields, since we want to give people the easy option first.